components/bsx/Create/CreateToken.vue

```diff
@@ -313,6 +313,11 @@ export default class CreateToken extends mixins(
       if (e instanceof Error) {
         showNotification(e.toString(), notificationTypes.danger)
         this.stopLoader()
+
+        if (retryCount < 3) {
+          showNotification('You will retry to mint NFT')
```

Comment:
sorry for being picky here but can we rephrase that
```suggestion
          showNotification('Retrying to complete minting process.')
```